content/workshops/openshift_4_101/lab5-rollbacks.md

```diff
@@ -33,18 +33,26 @@ Jump back to your OpenShift web console and let's add the webapp to our project.
 <blockquote>
 Click the "Add to Project" button
 </blockquote>
-<img src="../images/ocp-addToProjectButton.png" width="200"><br/>
+<img src="../images/ocp-addToProjectButton.png" width="70"><br/>
 <blockquote>
-Select the "From Git" tab and search for the nodejs builder image.
+Select the "From Git" icon, and click on the "Node.js" builder image.
 </blockquote>
+
+Make sure that you replace <b>YOUR_ACCOUNT</b> with your GitHub user ID. You will need to click on the "Show Advanced Git Options" expander to fill in the "Context Dir" field:
```

Comment:
Delete: "You will need to"  
Click the "Show Advanced Git Options" expander to fill in the "Context Dir" field.

Reply:
Done